scripts/globals/damage/physical_utilities.lua

```diff
@@ -0,0 +1,99 @@
+-----------------------------------
+-- Global, independent functions for physical calculations.
+-- Includes:
+-- fSTR, fSTR2
+-----------------------------------
+xi = xi or {}
+xi.combat = xi.combat or {}
+xi.combat.physical = xi.combat.physical or {}
+-----------------------------------
+
+-- "fSTR" in English Wikis. "SV function" in JP wiki and Studio Gobli.
+xi.combat.physical.calculateMeleeStatFactor = function(actor, target)
+    local fSTR = 0 -- The variable we want to calculate.
+
+    -- Calculate statDiff.
+    local statDiff     = actor:getStat(xi.mod.STR) - target:getStat(xi.mod.VIT)
+    local weaponRank   = actor:getWeaponDmgRank()
+    local statLowerCap = (7 + weaponRank * 2) * -2
+    local statUpperCap = (14 + weaponRank * 2) * 2
+
+    statDiff = utils.clamp(statDiff, statLowerCap, statUpperCap)
+
+    -- Calculate fSTR based on stat difference.
+    if statDiff >= 12 then
```

Comment:
Every condition here is divided by 4.  This could be done afterwards for simplicity